dojo/tools/semgrep/parser.py

```diff
@@ -67,5 +67,7 @@ def convert_severity(self, val):
             return "Low"
         elif "ERROR" == val.upper():
             return "High"
+        elif "INFO" == val.upper():
+            return "Informational"
```

Comment:
`Informational` is not a correct value. we should have `Info` here.
```suggestion
            return "Info"
```